common/helpers.go

```diff
@@ -240,3 +242,22 @@ func asGojaValue(ctx context.Context, v any) goja.Value {
 func gojaValueToString(ctx context.Context, v any) string {
 	return asGojaValue(ctx, v).String()
 }
+
+// readFile reads a file from FS. If path is relative, it's
+// converted to absolute path based on current working dir.
+func readFile(path string) (string, error) {
```

Comment:
Actually, i think it might be best to drop the readFile functionality. It will be difficult to work with in the cloud. We can add it back in if customer's ask for it. It's also a security risk since it looks like we can read any (permissions granted) file from the filesystem.